site/_js/analytics.js

```diff
@@ -0,0 +1,156 @@
+import {
+  onCLS,
+  onFCP,
+  onFID,
+  onLCP,
+  onTTFB,
+  onINP,
+} from 'web-vitals/attribution';
+import {dimensions} from '../_data/analytics.json';
+
+// Events missing from DevSite include:
+//   * "devsite-analytics-observation", generated by the Metric class for page load timing
+//   * "devsite-analytics-error", wired up to the onerror handler
+//   * "devsite-analytics-set-dimension", for user response to custom questions
+//   * "devsite-analytics-scope", which constructs complex click events (was used for header)
+//
+// Note that many parts of our code are annotated with an "gc-analytics-event" attribute, but this
+// is actually ignored in DevSite v2. Instead, any links that have `data-category` automatically
+// have clicks logged (see below).
+
+function getAnalyticsDataFromElement(elem, defaultAction = 'click') {
```

Comment:
I think we probably don't want to include this function, especially since this code will be temporary, and the functionality isn't being used to right now.

Reply:
Yeah I was torn between consistency, and just installing what this PR needs.

Removed them all now.